simulatingrisk/hawkdove/model.py

```diff
@@ -106,6 +103,9 @@ def play(self):
         # update total points based on payoff this round
         self.points += payoff
 
+        # store this round's choice as previous choice
+        self.last_choice = self.choice
+
```

Comment:
now setting in the `play` stage instead of in the `choose` so that last choice will be populated before _any_ agents start choosing for the next round